modules/tinymce/src/core/test/ts/browser/dom/SerializerTest.ts

```diff
@@ -647,7 +647,7 @@ describe('browser.tinymce.core.dom.SerializerTest', () => {
   });
 
   it('BR at end of blocks', () => {
-    const ser = DomSerializer({ fix_list_elements: true });
+    const ser = DomSerializer({ fix_list_elements: true, remove_trailing_brs: true });
```

Comment:
Does adding this cover all the test that were removed from DomParserTest?

Reply:
I added the removed test to SerializerTest file